### ISSUE TITLE
Improving accuracy

### ISSUE THREAD

**Issue by Zumbalamambo** [davidsandberg/facenet#889]
I used the following hyperparams to train on casia.

`python src/train_tripletloss.py --logs_base_dir ./logs --models_base_dir ./models --data_dir facenet/data/casia_maxpy_mtcnnpy_182  --image_size 160 --model_def models.inception_resnet_v1  --optimizer RMSPROP --learning_rate 0.01 --weight_decay 1e-4 --max_nrof_epochs 600 --gpu_memory_fraction 0.1 
`

The Embedding is 512 dimensions. people_per_batch is 6 and batch_size is 9 and images_per_person is also 6. 

However after training the model , i tried to validate on lfw.

This is the output on validation.

> Accuracy: 0.76133+-0.01545
> Validation rate: 0.07233+-0.01984 @ FAR=0.00100
> Area Under Curve (AUC): 0.845
> Equal Error Rate (EER): 0.238

why is my accuracy just 76 %? How do I improve it?

**Comment by test4fest** [comment]
Did you aligned the LFW dasaset before running validation?
would you provide more detail about (code) for validaion part?

**Comment by yuwgle** [comment]
try to increase batch_size
